fedgov-cv-rgaencoder/src/test/java/gov/usdot/cv/rgaencoder/RGAEncodeTest.java

```diff
@@ -25,20 +25,47 @@
 import static org.mockito.Mockito.when;
 
 import gov.usdot.cv.mapencoder.ByteArrayObject;
+import gov.usdot.cv.mapencoder.Position3D;
 
 public class RGAEncodeTest {
     private static final Logger logger = LogManager.getLogger(RGAEncodeTest.class);
     Encoder encoder;
     RGAData mockRGA;
+    BaseLayer mockBaseLayer;
```

Comment:
No need to update this as we do not have the JNI wrapper code to test this yet. Guess you can leave it like this for now.